test/unit/test_blas_tile/test_add.h

```diff
@@ -0,0 +1,65 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2024, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <sstream>
+
+#include <dlaf/blas/enum_output.h>
+#include <dlaf/blas/tile.h>
+#include <dlaf/blas/tile_extensions.h>
+#include <dlaf/matrix/tile.h>
+
+#include <gtest/gtest.h>
+
+#include <dlaf_test/blas/invoke.h>
+#include <dlaf_test/matrix/util_generic_blas.h>
+#include <dlaf_test/matrix/util_generic_blas_extensions.h>
+#include <dlaf_test/matrix/util_tile.h>
+#include <dlaf_test/matrix/util_tile_blas.h>
+#include <dlaf_test/util_types.h>
+
+namespace dlaf {
+namespace test {
+
+// Computes A += alpha * B
+
```

Comment:
Comment out of place. Moreover, it looks superfluous and a bit misleading (it computes, but just for checking the result correctness)